Configuration/StandardSequences/python/DigiToRaw_Repack_cff.py

```diff
@@ -72,7 +72,7 @@
                                         cms.InputTag('siStripZeroSuppressionHLT','ScopeMode')),
                                 )
 
-from RecoLocalTracker.SiStripClusterizer.SiStripClusters2ApproxClusters_cff import hltSiStripClusters2ApproxClusters
+from RecoLocalTracker.SiStripClusterizer.SiStripClusters2ApproxClusters_cff import * 
 
 from EventFilter.Utilities.EvFFEDExcluder_cfi import EvFFEDExcluder as _EvFFEDExcluder
 rawPrimeDataRepacker = _EvFFEDExcluder.clone(
```

Comment:
As privately discussed we need to put `onlineBeamSpotProducer`in the task below `DigiToApproxClusterRawTask` to avoid using the default (0,0,0) beamspot, see [log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-e01bcc/33319/runTheMatrix-results/140.58_RunHI2018/step2_RunHI2018.log)
Notice that might require repacking the SCAL FED as well for Run-2, and changing the input tags in the producer.